test/coverageParser.js

```diff
@@ -2,8 +2,11 @@
     'use strict';
 
     var expect = helper.chai.expect;
-    var validFormats = ['lcov'];
-
+    var validFormats = ['lcov','jacoco'];
+    var errorMsg = new Map([
+        {lcov: 'Failed to parse string'},
+        {jacoco: 'Request handle not found.'}
```

Comment:
The previous message was not very good. But now this one seems a bit bad for the user to understand that the files contents are bad, or that the file does not exist.
Can you provide a nicer message when this cases happen?

Reply:
Unfortunately, the error messages are generated by other dependent modules. We would have to create pull requests to those libraries to change the message.

Reply:
But publishing this with that message is quite confusing.
At least we could check if the file exists and give one error before invoking the parser.
And then if the parser fails change the message to something better like could not parse.

Reply:
I have added a prefix 

> Failed to parse jacoco report

 to the error message for better understanding. This would help us to understand that the report could not be parsed and show the error from the dependent library.